tests/test_util.py

```diff
@@ -152,4 +149,6 @@ def test_convert_to_stripe_object_and_back(self):
 
     def test_sanitize_id(self):
         sanitized_id = util.sanitize_id("cu  %x 123")
+        if isinstance(sanitized_id, bytes):
```

Comment:
Where did we start getting bytes from?

Reply:
Good catch, this really shouldn't be necessary. `sanitize_id` calls `quote_plus` which returns a string.